cdk/kraken/src/deploy.ts

```diff
@@ -64,12 +78,23 @@ export class DeployJob extends CheckoutJob {
           },
         },
         {
-          name: "Deploy",
-          run: dedent`aws eks --region us-east-1 update-kubeconfig --name production --role-arn arn:aws:iam::\${AWS_ACCOUNT_ID}:role/kubectl
+          name: 'Push to kube-manifests repository',
+          run: dedent`cd kube-manifests
+          mkdir -p \${{ github.repository }}
```

Comment:
Should we make a directory with `github.repository` (so like a folder pennlabs then a subfolder penn-courses), or should we just make a directory with the `RELEASE_NAME`? 

I feel like the latter might make more sense since everything will be prefixed by pennlabs anyways.